schedulables/ReservationAvailabilityPoller.js

```diff
@@ -0,0 +1,78 @@
+const enmap = require('enmap')
+const discord = require('discord.js')
+const rezClient = require('../modules/CampingReservationClient')
+var _ = require('underscore')
+const { conf } = require('../commands/dota')
+const TableBoi = require("../modules/TableBoi");
+
+// reservations from previous run, used to figure out the new reservations
+const previousReservations = new Map()
```

Comment:
persistent store in next iteration?

Reply:
good idea

Reply:
Should I use the same enmap or a separate one?